janusgraph-test/src/test/java/org/janusgraph/graphdb/serializer/CustomHashMapSerializer.java

```diff
@@ -0,0 +1,82 @@
+// Copyright 2017 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.janusgraph.graphdb.serializer;
+
+import org.janusgraph.core.*;
+import org.janusgraph.core.schema.JanusGraphManagement;
+import org.janusgraph.diskstorage.configuration.ModifiableConfiguration;
+import org.janusgraph.graphdb.configuration.GraphDatabaseConfiguration;
+import org.janusgraph.graphdb.database.StandardJanusGraph;
+import org.janusgraph.graphdb.serializer.attributes.*;
+import org.apache.tinkerpop.gremlin.structure.Vertex;
+import org.junit.*;
+
+import static org.junit.Assert.*;
+
+import java.util.HashMap;
+
+/**
+ *  @author Scott McQuillan (scott.mcquillan@uk.ibm.com)
+ */
+public class CustomHashMapSerializer {
+
```

Comment:
Didn't seem appropriate to add as an extra test to [SerializerTest](https://github.com/JanusGraph/janusgraph/blob/cb9c06aeb8ab95c3a4c4cc7575b7ad97ff79ddca/janusgraph-test/src/test/java/org/janusgraph/graphdb/serializer/SerializerTest.java) as that class wasn't creating any instances of `StandardJanusGraph`. But the [SerializerGraphConfiguration](https://github.com/JanusGraph/janusgraph/blob/cb9c06aeb8ab95c3a4c4cc7575b7ad97ff79ddca/janusgraph-test/src/test/java/org/janusgraph/graphdb/serializer/SerializerGraphConfiguration.java) unit tests was and it seemed a more suitable place to add the new `testCustomHashMapSerializer` so I've moved it into there